src/Db/db.rb

```diff
@@ -0,0 +1,75 @@
+require 'open-uri'
+require 'openssl'
+require 'open3'
+PKSIM_DB_FILE_NAME = 'PKSimDB.sqlite'
+
+namespace :db do
+  desc "Performs a dump of the database"
+  task :dump do
+    puts "Starting database dump..."
+    sh(sqlite3, "#{pksimDb}", ".output #{dump_file}", ".dump", ".exit" )
```

Comment:
each . command with sqlite3 can be executed one after the other that way

Reply:
interesting